### DISCUSSION TITLE
Verwijderen gebruikers?

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#259] [Q&A] [answered]
Klopt het dat ik geen gebruikers kan verwijderen als administrator? Ook als gebruiker zelf kan ik mijn account geloof ik niet verwijderen (in de demo-omgeving).
![image](https://user-images.githubusercontent.com/79267472/140375646-6a3312ac-e7ee-44a2-9375-7a75cb8e51cf.png)

**Comment by marcvanandel** [comment, marked as answer]
Euh ... Nee, dat kan niet 😬 Je kan jezelf niet verwijderen als gebruiker omdat je account een koppeling is naar je officiële Office365 account (of andere identity provider) Jezelf verwijderen zou betekenen dat je de koppeling wil verwijderen ... dat zou mogelijk nieuwe functionaliteit zijn.

Het is wel mogelijk om een `Organization` te verlaten. Dat zit onder de knop 'Danger Zone' (of NL: Gevarenzone). We hebben met de laatste release van afgelopen maandag ook opgeleverd dat je een Organization kunt verwijderen. Dat kan alleen als deze Org geen sensoren (devices eigenlijk) meer heeft.

Het beheer van gebruikers als admin ... is waarschijnlijk nog veel te weinig / te kort ... maar requirements hiervoor ontbreken nog. Hoe zou dat moeten werken? En zou dat in SensRNet Registry Node moeten gebeuren ... of in het gekoppelde identity provider systeem? Dit laatste is een veel gebruikt model. Daarom hebben we dit niet verder uitgebreid gebouwd. Maar precieze details over hoe deze functionaliteit gewenst is, ontbreekt nog volledig. Vandaar een enigszins gebrekkige werking (terwijl toch rijk genoeg om mee te kunnen werken, denk ik 😉 )
